tests/run.bats

```diff
@@ -227,6 +227,16 @@ function configure_and_check_user() {
 	expect_output --substring "unknown user" "run as unknown user"
 }
 
+@test "run --env" {
+	skip_if_no_runtime
+
+	_prefetch alpine
+	run_buildah from --quiet --pull=false --signature-policy ${TESTSDIR}/policy.json alpine
+	cid=$output
+	run_buildah run --env foo=foobar $cid -- /bin/sh -c 'echo $foo'
+	expect_output "foobar"
+}
```

Comment:
Could you add a test to commit the image and then inspect it for foo environment variable?  IE Make sure it does not get committed to the image.